kivy/uix/modalview.py

```diff
@@ -172,11 +173,13 @@ class ModalView(AnchorLayout):
 
     # Internals properties used for graphical representation.
 
-    _anim_alpha = NumericProperty(0)
+    _anim_alpha = NumericProperty()
```

Comment:
Please put back the default property values. There's no reason to remove them, so removing them can only contribute to bugs and it helps with code clarity being able to see the default value, even if we know what it defaults to when not provided. Same for all the other places.

Reply:
Ok